src/types/types.ts

```diff
@@ -1,9 +1,10 @@
 import type {
-  SearchResponse as MeiliSearchResponse,
   FacetDistribution,
+  SearchResponse as MeiliSearchResponse,
 } from 'meilisearch'
-import type { SearchClient } from 'instantsearch.js'
+
 import type { MultipleQueriesQuery as AlgoliaMultipleQueriesQuery } from '@algolia/client-search'
+import type { SearchClient } from 'instantsearch.js'
```

Comment:
I don't the order of import needs to be changed, could you rollback?

Reply:
The import order has been reverted.